doc/_scripts/software_maturity/software_maturity_features.yaml

```diff
@@ -102,15 +102,30 @@ features:
     Monitor Mode:
       rule: WIFI_NRF700X && NRF700X_RAW_DATA_RX
       boards_and_shields:
+        - SHIELD_NRF7002EK
+        - SHIELD_NRF7002EK_NRF7000
+        - SHIELD_NRF7002EK_NRF7001
+        - SHIELD_NRF7002EB
         - BOARD_NRF7002DK_NRF5340_CPUAPP
+        - BOARD_NRF7002DK_NRF7001_NRF5340_CPUAPP
     TX injection Mode:
       rule: WIFI_NRF700X && NRF700X_RAW_DATA_TX
       boards_and_shields:
+        - SHIELD_NRF7002EK
+        - SHIELD_NRF7002EK_NRF7000
+        - SHIELD_NRF7002EK_NRF7001
+        - SHIELD_NRF7002EB
         - BOARD_NRF7002DK_NRF5340_CPUAPP
+        - BOARD_NRF7002DK_NRF7001_NRF5340_CPUAPP
     Promiscuous Mode:
       rule: WIFI_NRF700X && NRF700X_PROMISC_DATA_RX
       boards_and_shields:
+        - SHIELD_NRF7002EK
+        - SHIELD_NRF7002EK_NRF7000
```

Comment:
There is no STA support in 7000, it's scan-only, so, promiscuous should also be removed?